tools/emptyDropsWrapper/test/prepRDS.R

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env Rscript
+library(nbHelpers)
```

Comment:
How about renaming `nbHelpers` to something more descriptive?

Reply:
nit: `library('nbHelpers')` to be consistent to other import usages

Reply:
Since the only function of `nbHelpers` being used here is `read10xMatrix()`, I took a look at the https://github.com/barkasn/nbHelpers, and wonder if it is feasible to move the `read10xMatrix()` code snippets here living with the other scripts? Otherwise if we need to use this `emptyDropsWrapper` in our pipelines, we need to do code review on the nbHelpers package. This also makes me think if we should make a new repo of Rscripts/Rpacakges besides `sctools` in HCA Github Org. Opinions? @samanehsan @barkasn

Reply:
I addressed this by copying read10Xmatrix into the test script directly.

Couple of points:

* nbHelpers is my personal git repository code for small reusable pieces of code, so it can't be renamed. We should never link to this repos as Rex points out and I should have removed this. 

* In R both library('nbHelpers') and library(nbHelpers) are perfectly valid, but the quoted version is much better style and as Rex points out consistent with everything else. Its been completely removed in this case

* R package repos have a very specific structure and although you can put multiple packages in a git repo it breaks convention and the expectation that the package can be installed with github_install() from the devtools package. We could make a single repo with all the hca helpers, but I don't think we have enough right now.